### PR TITLE
feat: adds iOS collect guide

### PR DESCRIPTION
**Pull request by brigonzalez**
## Description

- adds iOS collect guide

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- will add screenshots in a bit

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

**Review by lcschy — APPROVED** [review]

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+3. On the next screen save the project, wherever you'd like.
+
+This will launch Xcode with the newly created project.
+
+## Install the iOS Elements SDK
+
+We will need to install Basis Theory's [iOS Elements SDK](/docs/sdks/mobile/ios), which will render a secure `UITextField` for capturing the data.
+
+With the Xcode window in focus, select File > Add Packages. In the dialog that appears, enter "https://github.com/Basis-Theory/basistheory-ios" in the search field on the top right, then click Add Package on this screen and the next prompts.
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:50]: `"https://github.com/Basis-Theory/basistheory-ios"` is creating a weird link in the guide.

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+## Add Your Form Components
+
+Now we need to add [TextElementUITextField](/docs/sdks/mobile/ios/types#textelementuitextfield) iOS element component to our app.
+
+The easiest way to add a [TextElementUITextField](/docs/sdks/mobile/ios/types#textelementuitextfield) is to first drop a `UITextField` onto your `Main.storyboard` file. To do this open the `Main.storyboard` file, and click on plus button on the top right and search for "UITextField" in
+the dialog that appears. Now drag and drop the Text Field component anywhere on the iPhone screen. Feel free to size the `UITextField` to your liking.
+
+With the new `UITextField` selected, open up the Identity Inspector on the right and select "TextElementUITextField" from the class drop down, similar to the image below.
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:59]:
```suggestion
With the new `UITextField` selected, open up the Identity Inspector on the right and select `TextElementUITextField` from the class drop down, similar to the image below.
```

File: docs/guides/collect/collect-data-with-ios.mdx (hunk 1 above)
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:50]:
```suggestion
With the Xcode window in focus, select File > Add Packages. In the dialog that appears, enter `https://github.com/Basis-Theory/basistheory-ios` in the search field on the top right, then click Add Package on this screen and the next prompts.
```

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+Clicking on the button above will open the `Main.storyboard` on a new pane on the right. Now in the Project Navigator on the left, click on the `ViewController.swift` file. This should replace the `Main.storyboard` file on the right pane.
+
+Next, we control-drag from the `TextElementUITextField` in `Main.storyboard` to the top of our `ViewController` class in the `ViewController.swift` file on the right. In the window that appears, type in "ssnTextElement" for the
+Name and click Connect. Then Xcode should suggest you import BasisTheoryElements, so let's go ahead and do that now.
+
+You should have something similar to the following for `ViewController.swift` after these steps.
+
+``` swift showLineNumbers title="Collect iOS Guide/ViewController.swift"
+//
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:80]: Remove these comments for your examples.

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        // Do any additional setup after loading the view.
+    }
+}
+
+```
+
+We'll use the new `ssnTextElement` outlet to style our new element. Let's write some statements to style our component.
+
+``` swift showLineNumbers title="Collect iOS Guide/ViewController.swift"
+//
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:104]: Remove these comments for your examples.

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+
+As you can see above, we take advantage of the existing [UITextField](https://developer.apple.com/documentation/uikit/uitextfield) API to style our component. All of our iOS elements extend
+[UITextField](https://developer.apple.com/documentation/uikit/uitextfield) for ease of use and speed to production.
+
+## Add a Mask to Text Element
+
+Let's add a `mask` to format the input as the user types in the text field. The following sets a mask that looks like a social security number (ie. ###-##-####).
+
+``` swift showLineNumbers title="Collect iOS Guide/ViewController.swift"
+//
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:142]: Remove these comments for your examples.

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+We need to add a tokenize `UIButton` and a function to handle the tokenization request. Open the `Main.storyboard` file, and click on plus button on the top right and search for "UIButton" in the dialog that appears and drag
+and drop the Filled Button component anywhere under our `TextElementUITextField`. Let's change the name of this button to "Tokenize" by double-clicking on the new UIButton.
+
+Now we need to add an action for this new button. Follow the previous steps from [Style Your Form Components](#style-your-form-components) to get `Main.storyboard` and `ViewController.swift` files side-by-side. Control-drag
+the "Tokenize" UIButton into `ViewController.swift`, then choose Action for the Connection and "tokenize" for the Name.
+
+Now we'll add the following code to tokenize the data with Basis Theory.
+
+```swift showLineNumbers title="Collect iOS Guide/ViewController.swift"
+//
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:202]: Remove these comments for your examples.

File: docs/guides/collect/collect-data-with-ios.mdx [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+## Conclusion
+
+You can now capture any data without your iOS app accessing the underlying value drastically reducing compliance and regulatory scope.
+
+Try typing in a social security number in the [TextElementUITextField](/docs/sdks/mobile/ios/types#textelementuitextfield) and tap on Tokenize. Basis Theory's iOS SDK will pass the value for the element reference. This will
+be securely submitted directly to the [Create Token Endpoint](/docs/api/tokens#create-token). The resulting [token](/docs/concepts/what-are-tokens) is printed to the Xcode console.
+
+## Learn More
+
+- [Query User Data from an API](/docs/blueprints/personal-information/query-user-data-from-api)
```
Comment (bweber) [inline docs/guides/collect/collect-data-with-ios.mdx:279]: We should send them to a Share guide next instead. Maybe https://developers.basistheory.com/docs/guides/share/send-data-to-third-party

**Review by bweber — CHANGES_REQUESTED** [review]

File: docs/guides/collect/collect-data-with-ios.mdx (hunk 7 above)
Comment (brigonzalez) [reply docs/guides/collect/collect-data-with-ios.mdx:279]: lol sure. the choices here seemed random to me

**Pull request merged**